### PR TITLE
Update Nomad_Moogle.lua for multiple limit break quests beyond 75

### PR DESCRIPTION
**Pull request by TeoTwawki**
Credit lies with Fatalerror66 and @AlexandreCA I am only doing this because Alex needs to learn2git...

Alex assures me this works. I have not personally tested, and have no plans to - I am short on free time.

### CODE REVIEW

**Comment by AlexandreCA** [comment]
thanks a lot @TeoTwawki . I assures this works. I tested that on my server. :)

File: scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua [hunk 1]
```diff
@@ -1,57 +1,137 @@
...
+function onTrigger(player,npc)
+    if (player:hasKeyItem(LIMIT_BREAKER) == false and player:getMainLvl() >= 75) then
+        player:startEvent(10045,75,2,10,7,30,302895,4095);
+    elseif (player:getMainLvl() == 75 and player:levelCap() == 75 and MAX_LEVEL >= 80 and player:getQuestStatus(JEUNO,NEW_WORLDS_AWAIT) == QUEST_AVAILABLE) then
+        player:startEvent(10045,0,1,1,0);
+    elseif (player:getMainLvl() >= 76 and player:levelCap() == 80 and MAX_LEVEL >= 85 and player:getQuestStatus(JEUNO,EXPANDING_HORIZONS) == QUEST_AVAILABLE) then
+        player:startEvent(10045,0,1,2,0);
+    elseif (player:getMainLvl() >= 81 and player:levelCap() == 85 and MAX_LEVEL >= 90 and player:getQuestStatus(JEUNO,BEYOND_THE_STARS) == QUEST_AVAILABLE) then
+        player:startEvent(10045,0,1,3,0);
+    elseif (player:getMainLvl() >= 91 and player:levelCap() == 90 and MAX_LEVEL >= 95 and player:getQuestStatus(JEUNO,DORMANT_POWERS_DISLODGED) == QUEST_AVAILABLE) then
```
Comment (xdemolish) [inline scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua]: wat

Comment (TeoTwawki) [inline scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua]: @xdemolish @AlexandreCA Guessing that should have been 86

**Comment by TeoTwawki** [comment]
Amended for what I presume was a copy-paste error.

File: scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua (hunk 1 above)
Comment (AlexandreCA) [inline scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua]: 86 for?

Comment (TeoTwawki) [inline scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua]: weeeell you can't be level 91 if your cap was 90 :smile:

**Comment by AlexandreCA** [comment]
haha. http://ffxiclopedia.wikia.com/wiki/Dormant_Powers_Dislodged

The problem is, I don't change nothing. You have access for the modification.

File: scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua (hunk 1 above)
Comment (AlexandreCA) [inline scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua]: haha. http://ffxiclopedia.wikia.com/wiki/Dormant_Powers_Dislodged

The problem is: I can do change anything. You have access for the modification.

Comment (TeoTwawki) [inline scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua]: I already did it hours ago

**Pull request merged**